sdk/src/main/java/dev/cdevents/spec/schemas/SchemaLoader.java

```diff
@@ -37,7 +38,9 @@ public static String loadSchema(String schema) {
             }
 
             // load from filesystem next
-            return Files.readString(Paths.get(CDEventConstants.SCHEMA_FOLDER + "/" + schema));
+            try (Stream<String> lines = Files.lines(Paths.get(CDEventConstants.SCHEMA_FOLDER + "/" + schema))) {
+                return lines.collect(Collectors.joining("\n"));
+            }
```

Comment:
I guess the exception will surface to the outer `catch` - what is the value of the extra `try` here?

Reply:
The `try` block is used to auto close the stream.